app/views/admin/tutorials/new.html.erb

```diff
@@ -1,18 +1,37 @@
-<h2>New Tutorial</h2>
-<%= form_for [:admin, @tutorial] do |f| %>
-  <%= f.label :title %>
-  <%= f.text_field :title, class: "block col-4 field" %>
-  <%= f.label :description %>
-  <%= f.text_area :description, class: "block col-4 field"  %>
-  <%= f.label :thumbnail %>
-  <%= f.text_field :thumbnail, class: "block col-4 field"  %>
+<div id="new-tutorial-form">
+  <h2>New Tutorial</h2>
+  <%= form_for [:admin, @tutorial] do |f| %>
+    <%= f.label :title %>
+    <%= f.text_field :title, class: "block col-4 field" %>
+    <%= f.label :description %>
+    <%= f.text_area :description, class: "block col-4 field"  %>
+    <%= f.label :thumbnail %>
+    <%= f.text_field :thumbnail, class: "block col-4 field"  %>
 
-  <h3>Videos</h3>
+    <h3>Videos</h3>
 
-  <%= f.fields_for :videos do |builder| %>
-    <%= f.label :youtube_url %>
-    <%= f.text_field :video_id, class: "block col-4 field" %>
+    <%= f.fields_for :videos do |builder| %>
+      <%= f.label :youtube_url %>
+      <%= f.text_field :video_id, class: "block col-4 field" %>
+    <% end %>
+    <%= f.submit "Save", class: "mt2 btn btn-primary mb1 bg-teal" %>
   <% end %>
+</div>
+  <div data-controller="tutorials">
+    <%= link_to 'Import Youtube Playlist', "#", "data-action" => "click->tutorials#showPlaylistForm", class: "video-position-submit btn btn-outline", id: @tutorial.id %>
+  </div>
 
-  <%= f.submit "Save", class: "mt2 btn btn-primary mb1 bg-teal"  %>
-<% end %>
+  <div class="mt1" id="import-playlist-form">
+    <h3>Import New Playlist</h3>
```

Comment:
this one is hidden behind a link and will show if the link is clicked, it routes to a separate controller, tutorial_import